.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: "Deploy site"
+
+on:
+  pull_request:
+    branches:
+    - '*'
+  push:
+    branches:
+    - 'master'
+
+jobs:
+  example-1:
+    name: Deploy lektor website
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10'
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+          architecture: 'x64'
+      - name: Install dependencies
+        run: |
+          pip install -U pip
+          pip install "werkzeug<1.0"
+          pip install git+https://github.com/lektor/lektor#egg=Lektor
```

Comment:
It would be nice to eat our own dogfood and use a released version (preferably, the latest released version) of Lektor to build the docs.
Perhaps wait to fix this auto-deployment script until Lektor 4.0 is released, then just use that.  That should obviate the need for all the explicit pins.

Reply:
I switched it to using the latest release.